File: Ago-24-2021/C/ex004.c
#include<stdio.h>

int main(){
    float n1,n2,total;
    int n3;
    
    printf("No. do vendedor: ");
    scanf("%d", &n3);
    printf("Salario fixo do vendedor: ");
    scanf("%f", &n1);
    printf("Qual o valor em vendas: ");
    scanf("%f", &n2);

    total = n1 + (n2 * 0.15);

    printf("\nA comissao será: R$%.2f", n2 * 0.15);
    printf("\nO salario final é: R$%.2f", total);
    return 0;
}

File: _java/ex004.java
// Programa de cálculo de comissão para um vendedor
package Uniso._java;
import java.util.Scanner;

public class ex004 {
    public static void main(String[] args) {
        Scanner scanf = new Scanner(System.in);
        double salario, vendas, total;
        String n;

        System.out.print("Qual o nome do vendedor? ");
        n = scanf.nextLine();
        System.out.print("Qual o salário fixo de "+ n +"? ");
        salario = scanf.nextDouble();
        System.out.print("Qual o valor em vendas? ");
        vendas = scanf.nextDouble();

        total = salario + (vendas * 0.15);

        System.out.println("\nA comissão será de "+ (vendas * 0.15));
        System.out.println("O salário final de "+ n + " é igual a R$" + total);

        scanf.close();
    }
}
